gradio/networking.py

```diff
@@ -36,7 +36,7 @@
 ASSOCIATION_PATH_IN_STATIC = "static/apple-app-site-association"
 ASSOCIATION_PATH_IN_ROOT = "apple-app-site-association"
 
-FLAGGING_DIRECTORY = 'static/flagged/'
+FLAGGING_DIRECTORY = 'flagged/'
```

Comment:
This can be the default, but I would allow this to be overridden with a parameter in the Interface class.

Reply:
Added `flagging_dir` to `Interface`